pkg/minikube/bootstrapper/images/images.go

```diff
@@ -64,14 +74,53 @@ func componentImage(name string, v semver.Version, mirror string) string {
 	return fmt.Sprintf("%s:v%s", path.Join(kubernetesRepo(mirror), name), v)
 }
 
+func getLatestTag(url string, lastKnownGood string) string {
```

Comment:
comment here, also consider change name to something more meaningful, usually in golang word get is implied and not needed to be added to the name.

Reply:
also lets comment why we need this maybe link to an issue or describing what we are trying to solve here.

Reply:
fixed!